java/src/main/java/io/rapidpro/flows/definition/tests/text/ContainsAnyTest.java

```diff
@@ -55,17 +56,19 @@ public JsonElement toJson() {
         String[] rawWords = ExpressionUtils.tokenize(text);
 
         // run through each of our tests
-        List<String> matches = new ArrayList<>();
+        SortedSet<Integer> matches = new TreeSet<>();
         for (String test : tests) {
-            String match = testInWords(test, words, rawWords);
-            if (StringUtils.isNotEmpty(match)) {
-                matches.add(match);
-            }
+            findMatches(matches, test, words, rawWords);
```

Comment:
Will change Python to match, this was mostly because I didn't want to create an empty TreeSet object for every test.